plugins/WPML.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * ACF Pro Plugin.
+ *
+ * @package Junaidbhura\Composer\WPProPlugins\Plugins
+ */
+
+namespace Junaidbhura\Composer\WPProPlugins\Plugins;
+
+/**
+ * AcfPro class.
+ */
+class WPML {
```

Comment:
Change class block comment to reflect WPML plugin.

Rename the class and file name to CamelCase to reflect the other plugin class names.

```suggestion
/**
 * Wpml class.
 */
class Wpml {
```